components/Layout.js

```diff
@@ -0,0 +1,44 @@
+import styled from "styled-components";
+
+import { TitleBar } from "./TitleBar";
+
+export function Layout({ children }) {
+  return (
+    <Wrapper>
+      <TitleBar></TitleBar>
+      <StyledMain>{children}</StyledMain>
+      <Footer>
+        <Copyright>&copy; OM</Copyright>
+      </Footer>
+    </Wrapper>
+  );
+}
+
+const Wrapper = styled.div`
+  display: grid;
+  grid-template-rows: 80px auto 36px;
+  height: 100vh;
+`;
+
+const StyledMain = styled.main`
+  overflow-y: scroll;
+  margin: 0 auto;
+
+  ::-webkit-scrollbar {
+    width: 0;
+  }
+`;
+
+const Footer = styled.footer`
+  display: flex;
+  align-items: center;
+  justify-content: flex-end;
+`;
+
+const Copyright = styled.p`
+  margin: 0;
+  padding: 10px 7.5%;
+  font-size: 14px;
+  line-height: 16px;
+  color: rgba(26, 26, 26, 0.3);
```

Comment:
You should put that color value into custom properties as well as it is used multiple times

Reply:
Add all colors into custom properties.